README.md

```diff
@@ -3,12 +3,79 @@
 [![Read the Docs](https://img.shields.io/badge/docs-latest-blue.svg)][Documentation]
 
 
-CFFI
-====
+CFFI-ft
+=======
 
 Foreign Function Interface for Python calling C code.
 
-Please see the [Documentation] or uncompiled in the `doc/` subdirectory.
+## Why a fork?
+
+This is a fork of the CFFI project that adds support for free-threaded Python.
+Our intention is to upstream the changes in our fork but we are publishing this
+version to enable widespread real-world testing.
+
+### How to depend on CFFI-ft
+
+If your project depends on CFFI and you would like to publish free-threaded
+wheels or experiment with the free-threaded build, you can add a dependency on
+the `cffi-ft` on Python 3.13 and newer. For example, in a `pyproject.toml` file:
+
+```toml
+dependencies = [
+    ...,
+    "cffi; python_version<'3.13'",
+    "cfft-ft; python_version>='3.13'",
+]
+```
```

Comment:
Should probably mention build dependencies and specifying dependencies in a `setup.py` file too.

Additionally, before doing a real PyPI upload it's probably worth uploading cffi-ft to test.pypi.org and verify that the `python_version` environment markers actually work correctly for a test project across python versions with cffi and cffi_ft as both build and runtime dependencies.